### DISCUSSION TITLE
Support for Github Actions Webhook

### DISCUSSION THREAD

**Discussion by cupcakearmy** [directus/directus#9567] [Feature Requests]
Triggering github actions for building a site with directus as a common setup.
That means that generally you want to trigger a build whenever content is changed with a webhook.

Github actions are a bit weird with triggering actions and require some body parameters for webhooks.
https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event

One could do a custom hook, but it would be nice to have a custom payload/data field in the hooks interface.

**Comment by joselcvarela** [comment]
Hello @cupcakearmy 
This seems to be feasible by setting webhooks: https://docs.directus.io/configuration/webhooks/#webhooks
What do you think?

**Reply by cupcakearmy** [reply]
Nope, If you check the [paramteres documentation](https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event--parameters) there are required **body** parameters, that cannot be configured in the ui. You can set url and custom headers.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
